HHS/Head-Start-TTADP
# [Prod] Improvements to the TR and session forms, bug fixes, email test

## Description of change

- Rename the "COLLABORATOR_TRAINING_REPORTS" permission to "POC_TRAINING_REPORTS" to be a little clearer about what it does
- Update the titles for the TR and session forms to distinguish between them
- Update the button labels on those forms to be consistent across the site
- Added "end date in" code on the backend
- Improve error handling and detection when trying to delete a group that doesn't exist
- Fix the action menu being cut off on the Approved AR/My Alerts tables when there are very few entries
- Add a form to the admin section to allow for an email/Amazon SES test in prod
- Update event permissions to match newest understanding
- Update node.js from 18.16.0 to 18.16.1

## How to test

- Revisit the TR and session forms and confirm that they still work and the updates have been made, including the permission
- Apply an end date within the last 30 days filter and confirm that this works
- View the my alerts table with 1 item. Confirm you can see all the actions in the menu

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1730
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1885
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1917
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1923
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1924
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1927
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1936


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
